tlmgr-dump-tlpdb, tlmgr-recreate-tlpdb: add page (#8290)
tldr-pages/tldr: pages/common/tlmgr-recreate-tlpdb.md

# tlmgr recreate-tlpdb

> Recreate the TeX Live package database.

> > This command should be used with caution, because it is poorly documented.

Reviewer: Not completely sure this line should be here.

Author: Well, the command is clearly not meant to be used by end-users. I assume it's for developers only, which means there is no guarantee that it is safe to use in all cases, especially since you have to run it as root.
I'd say we either warn the user or we don't document the command at all.

Reviewer: Ok, I think the current wording is a little too negative, how do you feel about

```suggestion
> This command has a lack of documentation and should be used with caution. 
```

Reviewer 2: Both sounds good, @marchersimon which do you like more?

> > This command has no documentation and therefore should probably not be used by humans.

Reviewer: ```suggestion
> This command should not be invoked by the user. 
```

Author: Well, the thing is, I don't know if it shouldn't. I just assumed it, because there was absolutely no documentation, except for the source code itself.

Reviewer 2: I think this command exists to be used under certain circumstances, also there is a `--platform` argument.
 ```suggestion
```

Author: I wouldn't be so sure about that. There are even debug outputs in the source code. Also, this command has to be run as root, which can become risky if it isn't properly documented.

Reviewer 2: I just looked around for this command, and I found some threads where this command is the answer to some errors.

Author: How about
```suggestion
> This command should be used with caution, because it is poorly documented.
```

Reviewer 3: That looks like a good suggestion, @marchersimon
